storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -407,6 +407,23 @@ static bool check_server_version(unsigned long version_number,
     }
   }
 
+  std::string mysql_version = MYSQL_SERVER_VERSION;
```

Comment:
I would change the variable name to be more clear of what it is about. `mysql_version` seems like this is the server version we are trying to backup, while in fact this is pxb base mysql version.